packages/tailwind/src/tailwind.tsx

```diff
@@ -100,26 +101,23 @@ function getMediaQueryCSS(css: string) {
 
   let newCss = css
     .replace(mediaQueryRegex, (m) => {
-      return m.replace(
-        /([^{]+\{)([\s\S]+?)(\}\s*\})/gm,
-        (_, start, content, end) => {
-          const newcontent = (content as string).replace(
-            /(?:[\s\r\n]*)?(?<prop>[\w-]+)\s*:\s*(?<value>[^;\r\n]+)/gm,
-            (_, prop, value) => {
-              return `${prop}: ${value} !important`;
-            }
-          );
-
-          return `${start}${newcontent}${end}`;
-        }
-      );
+      return m.replace(/([^{]+\{)([\s\S]+?)(\}\s*\})/gm, (_, start, content, end) => {
+        const newContent = (content as string).replace(
+          /(?:[\s\r\n]*)?(?<prop>[\w-]+)\s*:\s*(?<value>[^};\r\n]+)/gm,
```

Comment:
`!important` was only being added on the last selector inside a media query because this regex was depending on finding a `;` that the tailwind-to-css lib doesn't output, so I added a `}` so it would stop. `(?<value>[^};`

I think the better fix would probably to fix the `twi` output to have semicolons, but this works for now and is captured in the tests.